src/CNCULT4.CPP

```diff
@@ -1352,10 +1352,10 @@ int RFI::rf_CheckAccessAndAlias(
 	int ret = 0;
 	if (ppMsg)
 		*ppMsg = strtmp("");	// insurance
-	char* fNameTry = strtemp( _MAX_PATH);
+	char* fNameTry = strtemp(FILENAME_MAX);
```

Comment:
@nealkruis, this potential usage of FILENAME_MAX is exactly why I would advocate for us to create a CSE-specific definition for MAX PATH, perhaps platform-dependent but on the order of Windows' _MAX_PATH. `strtemp` allocates space out of a static char array of size 400002. If FILENAME_MAX for any reason is defined as uint_max, that's a memory crash right there.

Reply:
This was solved by using a CSE define macro.